lib/node_modules/@stdlib/stats/base/dists/binomial/examples/index.js

```diff
@@ -18,7 +18,40 @@
 
 'use strict';
 
-var objectKeys = require( '@stdlib/utils/keys' );
 var binomial = require( './../lib' );
 
-console.log( objectKeys( binomial ) );
+/*
+* Let's take an example of rolling a fair dice 10 times and counting the number of times a 6 is rolled.
+* This situation can be modeled using a Binomial distribution with n = 10 and p = 1/6
+*/
+
+var n = 10;
+var p = 1/6;
+
+// Mean can be used to calculate the average number of times a 6 is rolled:
+console.log( binomial.mean( n, p ) );
+// => ~1.6667
+
+// PMF can be used to calculate the probability of getting a certain number of 6s (say 3 sixes):
+console.log( binomial.pmf( 3, n, p ) );
+// => ~0.1550
+
+// CDF can be used to calculate probability upto certain number of 6s (say upto 3 sixes):
+console.log( binomial.cdf( 3, n, p ) );
+// => ~0.9303
+
+// Quantile can be used to calculate the number of 6s at which you can be 80% confident that the actual number will not exceed.
+console.log( binomial.quantile( 0.8, n, p ) );
+// => 3
+
+// Standard deviation can be used to calculate the measure of the spread of 6s around the mean:
+console.log( binomial.stdev( n, p ) );
+// => ~1.1785
+
+// Skewness can be used to calculate the asymmetry of the distribution of 6s:
+console.log( binomial.skewness( n, p ) );
+// => ~0.5657
+
+// MGF can be used for more advanced statistical analyses and generating moments of the distribution.
```

Comment:
Let's use a colon here for consistency with the other comments:

```suggestion
// MGF can be used for more advanced statistical analyses and generating moments of the distribution:
```